src/hooks/item.ts

```diff
@@ -106,6 +107,46 @@ export default (
       });
     },
 
+    useItemsChildren: (
+      ids: UUID[],
+      options?: {
+        enabled?: boolean;
+        ordered?: boolean;
+        getUpdates?: boolean;
+        placeholderData?: List<Item>[];
+      },
+    ): UseQueryResult<List<Item>[]> => {
+      const enabled = options?.enabled ?? true;
+      const ordered = options?.ordered ?? true;
+
+      return useQuery({
+        queryKey: buildItemsChildrenKey(ids),
+        queryFn: () =>
+          Promise.all(
+            ids.map((id) =>
+              Api.getChildren(id, ordered, queryConfig).then((data) =>
+                List(data),
+              ),
+            ),
+          ),
+        onSuccess: async (items: List<Item>[]) => {
+          if (items.length) {
+            items.forEach((item) => {
+              if (item.size) {
+                item.forEach((elt) => {
+                  const { id } = elt;
+                  queryClient.setQueryData(buildItemKey(id), Map(elt));
```

Comment:
It's a bit confusing here, but I guess `item` is actually `children`. Maybe it would be interesting to also fill the `children` key.